docs/layers/grid-layer.md

```diff
@@ -0,0 +1,58 @@
+# GridLayer
+
+The GridLayer is a generic layer that can render heatmap. It takes latitude longitude delta
```

Comment:
Doc nits:
- render a grid-based heatmap.?
- top-left coordinate of each cell.?
- The grid cells can be given a height using the `getHeight` accessor and ?